src/Injector/InjectorInterface.php

```diff
@@ -55,4 +57,12 @@ public function inject($package, $type, IOInterface $io);
      * @return void
      */
     public function remove($package, IOInterface $io);
+
+    /**
+     * Set modules of the application.
+     *
+     * @param array $modules
+     * @return self
+     */
+    public function setApplicationModules(array $modules);
```

Comment:
Also, this interface is essentially an internal detail; I'm not sure if you can reliably extend plugins, nor that you'd want to, as then *both* would trigger. As such, changes to these interfaces is likely fine even after the 1.0 release. We *should* likely mark them as `@internal` at some point, though.